From 86375c270632d9fafa4ba52aeb8d4f373b9f46b4 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Tue, 20 May 2025 15:27:59 +0300
Subject: [PATCH] [SW-218875] g2: dt sensors as on chip

The TD are actual an "On Chip" sensors and not "On board" sensors.
As The TD stand for Temperature Diode, located on chip, but read/handle
by external on board component.

Relates to [SW-219718] g3: update sensors map

Change-Id: Ib5b43dfa2391b9c3fd6fe10cfc33806ebea686ca
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/src/gaudi2.c b/src/gaudi2.c
index 45f17b0..25503fd 100755
--- a/src/gaudi2.c
+++ b/src/gaudi2.c
@@ -79,10 +79,10 @@
 	HLML_TEMPERATURE_HBM,
 	HLML_TEMPERATURE_HBM,
 	HLML_TEMPERATURE_HBM,
-	HLML_TEMPERATURE_ON_BOARD,
-	HLML_TEMPERATURE_ON_BOARD,
-	HLML_TEMPERATURE_ON_BOARD,
-	HLML_TEMPERATURE_ON_BOARD,
+	HLML_TEMPERATURE_ON_AIP,
+	HLML_TEMPERATURE_ON_AIP,
+	HLML_TEMPERATURE_ON_AIP,
+	HLML_TEMPERATURE_ON_AIP,
 	HLML_TEMPERATURE_ON_BOARD,
 	HLML_TEMPERATURE_ON_BOARD,
 	HLML_TEMPERATURE_ON_BOARD,
